games-emulation/higan/higan-116_pre.ebuild

```diff
@@ -0,0 +1,164 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop toolchain-funcs xdg
+
+DESCRIPTION="A multi-system emulator focused on accuracy, preservation, and configurability"
+HOMEPAGE="https://github.com/higan-emu/higan"
+SRC_URI="https://github.com/higan-emu/${PN}/archive/nightly.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="
+	alsa ao +byuu +core-cv +core-fc +core-gb +core-gba +core-md +core-ms
+	+core-msx +core-n64 +core-ngp +core-pce +core-ps1 +core-sfc +core-sg
+	+core-ws +gtk higan-ui icarus openal +opengl oss +pulseaudio qt5 +sdl udev
+	xv"
+REQUIRED_USE="
+	^^ ( gtk qt5 )
```

Comment:
This will introduce a default conflict for every single `desktop` profile user. You don't want that.

Is there a preferred gui according to upstream?

Reply:
The user interface for higan is planned to be re-implemented some time in the future: https://github.com/higan-emu/higan/issues/28

Upstream believes the GTK2 backend is the most solid one right now for Linux systems, but it's best to not use that one because GTK2 support will definitely go away soon. So I think GTK3 should be the one we use here instead because it will be the most similar to the GTK2 backend used by higan in our past ebuild versions. I'll remove the qt5 USE flag then and build the GTK3 backend unconditionally.

Reply:
That doesn't mean you have to remove the Qt5 option altogether. You can drop the `qt5` flag and build it if the user has unset `gtk` instead.

Reply:
So we would have something like the following instead?
```
!gtk? ( qt5? (
		dev-qt/qtcore:5
		dev-qt/qtgui:5
		dev-qt/qtwidgets:5
	)
)
```

Reply:
yup.

Reply:
All right, that makes sense. I've gone ahead and removed the qt5 use and left the gtk use flag with a metadata entry to describe the new behavior.